### PR TITLE
Digibyte : Increase Compute Limits

### PR DESCRIPTION
**Pull request by abdulgig**
### Description

We need to increase the compute limits as the node requires more resources to operate 

### Related Issues

https://github.com/threefoldfoundation/www_threefold_marketplace/issues/38
https://github.com/threefoldfoundation/info_cloud/issues/15

### Checklist

- [ x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc

### CODE REVIEW

File: jumpscale/packages/vdc_dashboard/package.toml [hunk 1]
```diff
@@ -37,15 +36,12 @@ port = 5591
 
 [[servers]]
 name = "vdc_dashboard_root_proxy"
-domain = "vdc-dashboard.threefold.me"
+domain = "mazraa-dgbtests.vdctest.grid.tf"
```
Comment (xmonader) [inline jumpscale/packages/vdc_dashboard/package.toml:39]: please reset this file to how it was

**Review by xmonader — CHANGES_REQUESTED** [review]
package.toml shouldn't be part of changes

File: jumpscale/packages/vdc_dashboard/package.toml (hunk 1 above)
Comment (abdulgig) [reply jumpscale/packages/vdc_dashboard/package.toml:39]: Oh, wasn't modified by me, will see

Comment (abdulgig) [reply jumpscale/packages/vdc_dashboard/package.toml]: Restored the original version and it is excluded now

**Review by xmonader — APPROVED** [review]

**Pull request merged**